frontend/packages/console-shared/src/components/form-utils/StickySection.scss

```diff
@@ -0,0 +1,9 @@
+.ocs-sticky-section {
+  position: fixed;
+  width: 81%;
```

Comment:
Why is width 81%?

Reply:
Added 81% to match the width to the existing sections like builder section. i will update the value for small screens as well.
![image](https://user-images.githubusercontent.com/9964343/76829013-3dae7980-6848-11ea-9430-0a79ebcaea2d.png)

Reply:
@sahil143 I have updated it to 100% so that it stays same in all screens as well.

Reply:
@karthikjeeyar footer width is still covering the scrollbar.
![Screenshot from 2020-03-17 12-49-34](https://user-images.githubusercontent.com/9278015/76831736-e6130c80-684d-11ea-86d3-62c4771095d9.png)

Reply:
@karthikjeeyar Changing the position to sticky should fix this. That will keep the footer in form and fixed.

Reply:
Updated the position to sticky.